docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+letsencrypt:
+  build: .
+  ports:
+    - "443:443"
+  volumes:
+    - /etc/letsencrypt:/etc/letsencrypt/certs
```

Comment:
Other subdirectories will exist soon with the renewer. Do they need to be individually mapped?